signalbot/__init__.py

```diff
@@ -1,5 +1,5 @@
 from .bot import SignalBot
-from .command import Command, CommandError, triggered
+from .command import Command, CommandError, triggered, regex_triggered
```

Comment:
I'm going to write it here even though it doesn't below to this file.

Would you mind adding an example of this in the `examples/commands`?

Reply:
Done!